pkg/config/config_template.yaml

```diff
@@ -1485,6 +1485,15 @@ api_key:
 #
 # process_config:
 
+  {{- if (eq .OS "linux")}}
+  ## @param run_in_core_agent - custom object - optional
+  ## Specifies which agent collects process and/or container information (Linux only).
```

Comment:
Would be good to mention which agent can collect the information. They are the core agent and the process agent, right?

Reply:
I agree the description could be improved... I'm not sure "core agent" is clear to the customers by the way, maybe `datadog-agent process` would be clearer ?

Reply:
I made some improvements to better show that the flag is exclusive. In terms of specifying `datadog-agent process`, I think that also has some potential for confusion. I believe in containerized environments the process is just `agent`.